### PR TITLE
Feature/vumigo 142 remove hacky hack hack

### PR DESCRIPTION
**Pull request by jerith**
This is an extension of #42 that removes `hacky_hack_hack()` completely and replaces RedisManager with TxRedisManager.

### CODE REVIEW

File: go/apps/multi_surveys/vumi_app.py [hunk 1]
```diff
@@ -37,13 +30,14 @@ class MultiSurveyApplication(MamaPollApplication, GoApplicationMixin):
 
     def validate_config(self):
         self._go_validate_config()
+        self.redis_config = self.config.get('redis_manager')
```
Comment (smn) [inline go/apps/multi_surveys/vumi_app.py:33]: Do we need a default `{}` here?

**Comment by hodgestar** [comment]
I think there are some missing `yields` on `self.pm.save(...)`.

**Comment by jerith** [comment]
There aren't any tests around `self.pm.save()`, apparently. :-/

**Comment by smn** [comment]
I have a branch somewhere where I was working on that.

**Comment by hodgestar** [comment]
Can someone involved with surveys or multi_surveys push this along? Doesn't seem like it should be too difficult to land or declare no longer needed?

**Comment by jerith** [comment]
This is based on a very old version of vumi. See #117 for the new PR for this issue.

**Pull request closed without merging**